src/main/java/frc/robot/utilities/TunableNumber.java

```diff
@@ -27,10 +28,15 @@ public TunableNumber(
   public void periodic() {
     if (!Robot.isCompetition) {
       consumer.accept(numberEntry.getDouble(value));
+      value = numberEntry.getDouble(value);
```

Comment:
```suggestion
      value = numberEntry.getDouble(value);
      consumer.accept(value);
```

Have pity on Shuffleboard, don't ask for the same thing from the NetworkTable twice in the same cycle